cmd/dcs-web/templates/error.html

```diff
@@ -75,6 +75,7 @@
    <li><a href="./">Search</a></li>
    <li><a href="./about">About Code Search</a></li>
    <li><a href="./faq">FAQ</a></li>
+   <li><a href="./thirdparty">Third party projects</a></li>
```

Comment:
Instead of a top-level menu point, it might be better to reference this page from faq and/or about. Most users won’t care :)

Reply:
Ok, I'll do that.